sys/stat_example_test.go

```diff
@@ -0,0 +1,54 @@
+package sys_test
+
+import (
+	"io/fs"
+	"math"
+
+	"github.com/tetratelabs/wazero"
+	"github.com/tetratelabs/wazero/sys"
+)
+
+type sysStatFS struct{ fs.FS }
```

Comment:
@evacchi @GeorgeMac @ncruces I have to dash.. I spiked this just to put something out there. If I messed this up feel free to comment or correct. I'll fix anything including adding more details to RATIONALE.md sometime not tonight.

Reply:
Nice, this all tracks 👍